Makefile

```diff
@@ -50,17 +50,17 @@ help:
 	@echo " * 'install-containerd'	- Install customized containerd to system location"
 	@echo " * 'release'          	- Build release tarball"
 	@echo " * 'push'             	- Push release tarball to GCS"
-	@echo " * 'test'             	- Test cri-containerd with unit test"
-	@echo " * 'test-integration' 	- Test cri-containerd with integration test"
-	@echo " * 'test-cri'         	- Test cri-containerd with cri validation test"
-	@echo " * 'test-e2e-node'    	- Test cri-containerd with Kubernetes node e2e test"
+	@echo " * 'test'             	- Test cri with unit test"
+	@echo " * 'test-integration' 	- Test cri with integration test"
+	@echo " * 'test-cri'         	- Test cri with cri validation test"
+	@echo " * 'test-e2e-node'    	- Test cri with Kubernetes node e2e test"
 	@echo " * 'clean'            	- Clean artifacts"
 	@echo " * 'verify'           	- Execute the source code verification tools"
-	@echo " * 'proto'            	- Update protobuf of cri-containerd api"
+	@echo " * 'proto'            	- Update protobuf of the cri plugin api"
 	@echo " * 'install.tools'    	- Install tools used by verify"
-	@echo " * 'install.deps'     	- Install dependencies of cri-containerd (containerd, runc, cni) Note: BUILDTAGS defaults to 'seccomp apparmor' for runc build"
+	@echo " * 'install.deps'     	- Install dependencies of cri (Note: BUILDTAGS defaults to 'seccomp apparmor' for runc build")
```

Comment:
cri plugin

Reply:
same component is cri.. which is a plugin we don't say containerd executable.. every time.